src/logic/operate.js

```diff
@@ -0,0 +1,24 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let total;
+  if (operation === '+') {
+    total = Big(numberOne + numberTwo);
+  } else if (operation === '-') {
+    total = Big(numberOne - numberTwo);
+  } else if (operation === '/') {
+    total = Big(numberOne / numberTwo);
+  } else if (operation === '*') {
+    total = Big(numberOne * numberTwo);
+  } else if (operation === '%') {
+    if (numberTwo) {
+      total = Big((numberOne * 0.01) * numberTwo);
+    } else {
+      total = Big(numberOne * 0.01);
+    }
+  } else {
+    total = '';
```

Comment:
- [ ] Please also use Big.js operations here and not the regular math operations.